include/exec/static_thread_pool.hpp

```diff
@@ -251,7 +262,10 @@ namespace exec {
 
    public:
     static_thread_pool();
-    static_thread_pool(std::uint32_t threadCount, bwos_params params = {});
+    static_thread_pool(
+      std::uint32_t threadCount,
+      bwos_params params = {},
+      numa_policy* numa = get_numa_policy());
```

Comment:
i'm not a fan of raw pointers in public interfaces. why does `numa_policy` need to be dynamically polymorphic?

Reply:
Would you like a template parameter for the Numa policy more? There is actually no reason to use type erasure when all member functions are defined inline anyway. That way I wouldn't need to hard code an allocator, too.